src/app/dashboard/dashboard.component.html

```diff
@@ -8,21 +8,44 @@
 <div class="flex-container">
   <div class="balance">
     <div>Wykres</div>
-    <div>info</div>
+    <div>
+      <p>Godziny Przepracowane: {{ balanceData.hoursWorked }}</p>
+      <p>Minuty Przepracowane:{{ balanceData.minutesWorked }}</p>
+      <p>Godziny Do Przepracowania:{{ balanceData.hoursLeft }}</p>
+      <p>Minuty Do Przepracowania:{{ balanceData.minutesLeft }}</p>
+    </div>
   </div>
 </div>
 
 <div class="flex-container">
   <div class="start-registration">
     <div>Rozpocznij rejetrowanie pracy</div>
-    <button class="clock-in-button">
+    <button (click)="clockIn()" class="clock-in-button">
       <i class="material-icons">access_time</i>
     </button>
+
+    <p *ngIf="errorClockIn === 404">
+      Wystąpił błąd Error: {{ errorClockIn }} nie udało się zarejestrować
+      wejścia do pracy
+    </p>
```

Comment:
Założenie, że otrzymamy zawsze status 404 to bardzo optymistyczne założenie. A co jeśli dostaniemy jakiś inny status (np. 401 lub 500)? Wtedy użytkownik nie otrzyma żadnej informacji. Powinniśmy założyć, że w przypadku otrzymania statusu innego niż 200 powinna się wyświetlać ta informacja: 

```suggestion
    <p *ngIf="errorClockIn !== 200">
      Wystąpił błąd Error: {{ errorClockIn }} nie udało się zarejestrować
      wejścia do pracy
    </p>
```